README.md

```diff
@@ -1,56 +1,156 @@
-<picture>
+<a href="https://panel.holoviz.org/"><picture>
   <source media="(prefers-color-scheme: dark)" srcset="https://github.com/holoviz/panel/raw/main/doc/_static/logo_horizontal_dark_theme.png">
   <img src="https://github.com/holoviz/panel/raw/main/doc/_static/logo_horizontal_light_theme.png" alt="Panel logo -- text is white in dark theme and black in light theme" width=400/>
-</picture>
+</picture></a>
 
------------------
+# Panel is the most flexible data app framework for Python
```

Comment:
This strikes me as unnecessarily confrontational. Maybe:
```suggestion
# Panel: the most flexible data app framework for Python
```

That way I feel that it conveys aspiration, rather than making a challenge or claim that makes a reader say "oh yeah?" "is it really?".